docs/source/en/quicktour.mdx

```diff
@@ -318,74 +272,125 @@ The model outputs the final activations in the `logits` attribute. Apply the sof
 </tf>
 </frameworkcontent>
 
-<Tip>
 
-All 🤗 Transformers models (PyTorch or TensorFlow) outputs the tensors *before* the final activation
-function (like softmax) because the final activation function is often fused with the loss.
+All 🤗 Transformers models (PyTorch or TensorFlow) output the tensors *before* the final activation
+function (like softmax) because the final activation function is often fused with the loss. Model outputs are special dataclasses, so their attributes are autocompleted in an IDE. The model outputs behave like a tuple or a dictionary (you can index with an integer, a slice, or a string), and attributes that are `None` are ignored.
 
-</Tip>
+## Custom model builds
+
+You can modify the model's configuration class to change how a model is built. The configuration specifies a model's attributes, such as the number of hidden layers or attention heads. When you initialize a model from a custom configuration class, you are starting from scratch. The model attributes are randomly initialized, and you'll need to train the model before you can use it to get meaningful results.
+
+Start by importing a model's configuration class, and then you can change the number of attention heads, for instance:
 
-Models are a standard [`torch.nn.Module`](https://pytorch.org/docs/stable/nn.html#torch.nn.Module) or a [`tf.keras.Model`](https://www.tensorflow.org/api_docs/python/tf/keras/Model) so you can use them in your usual training loop. However, to make things easier, 🤗 Transformers provides a [`Trainer`] class for PyTorch that adds functionality for distributed training, mixed precision, and more. For TensorFlow, you can use the `fit` method from [Keras](https://keras.io/). Refer to the [training tutorial](./training) for more details.
+```py
+>>> from transformers import DistilBertConfig
+
+>>> my_config = DistilBertConfig(n_heads=12)
+```
+
+Create a model from your custom configuration:
+
+```py
+>>> from transformers import DistilBertForSequenceClassification
+
+>>> my_model = DistilBertForSequenceClassification(my_config)
+```
+
+Take a look at the [Create a custom architecture](./create_a_model) guide for more information about building custom configurations.
+
+## Trainer
```

Comment:
I don't think adding this section here helps. For one it's PyTorch only (and not delimited as such) and it's making the quick tour really long. I'd remove it.